bin/node-template/node/src/command.rs

```diff
@@ -71,11 +72,12 @@ pub fn run() -> sc_cli::Result<()> {
 		}
 		None => {
 			let runner = cli.create_runner(&cli.run)?;
-			runner.run_node(
-				service::new_light,
-				service::new_full,
-				node_template_runtime::VERSION
-			)
+			runner.print_node_infos(node_template_runtime::VERSION);
+			let service_fn = match runner.config().role {
+				Role::Light => service::new_light,
+				_ => service::new_full,
+			};
+			runner.run_node_until_exit(service_fn)
```

Comment:
(Maybe it shouldn't be called "service" anymore)

```suggestion
			runner.run_node_until_exit(
				|config| match config.role {
					Role::Light => service::new_light,
					_ => service::new_full,
				}
			)
```

Reply:
https://github.com/paritytech/substrate/pull/6352#issuecomment-644081379 Yeah, not sure what to call it now :sweat_smile:

Reply:
`if matches!(config.role, Role::Light) { ... } else { ... }` might be more elegant

Reply:
I think it's terse enough now